tools/deploy_translatable_strings.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# Script for pushing the translatable strings to qiskit-translations repo.
+
+# Non-travis variables used by this script.
+SOURCE_REPOSITORY="git@github.com:Qiskit/qiskit-machine-learning.git"
+SOURCE_DIR=`pwd`
+SOURCE_LANG='en'
+
+TARGET_REPOSITORY="git@github.com:qiskit-community/qiskit-translations.git"
+TARGET_BRANCH_PO="master"
+
+DOC_DIR_PO="docs/locale/"
+
+echo "show current dir: "
+pwd
+
+pushd docs
+
+# Extract document's translatable messages into pot files
+# https://sphinx-intl.readthedocs.io/en/master/quickstart.html
+echo "Extract document's translatable messages into pot files and generate po files"
+tox -egettext -- -D language=$SOURCE_LANG
+
+echo "Setup ssh keys"
+pwd
+set -e
+# Add qiskit-translations push key to ssh-agent
+# TODO: Update me!
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/Qiskit/qiskit-machine-learning/pull/130/commits/ae168a8cf56a2d53a7a8031ea4a02039eec9439e